test/TestData/TestData.jl

```diff
@@ -10,12 +10,33 @@ module TestData
 using OpticSim
 using OpticSim: tobeziersegments # try to use only exported functions so this list should stay short
 using OpticSim.Geometry
+using OpticSim.Emitters
 using StaticArrays
 using DataFrames
 using Unitful
 using Images
 using Base: @.
 using LinearAlgebra
+using BenchmarkTools
+
+
+### Emitters
+
+function RectGridAllocations()
```

Comment:
100% agree on adding a test for Emitter allocations to ensure this kind of regression doesn't happen again - but it should really go in [Benchmarks.jl](https://github.com/microsoft/OpticSim.jl/blob/main/test/Benchmarks/Benchmarks.jl) and be added to [`all_benchmarks`](https://github.com/microsoft/OpticSim.jl/blob/9eec4c489c958847c55c83c01c559b614ab87b64/test/Benchmarks/Benchmarks.jl#L67) which will then be tested as part of the [allocations tests](https://github.com/microsoft/OpticSim.jl/blob/main/test/testsets/Allocations.jl)